querybook/webapp/components/DataDocStatementExecution/StatementResult.tsx

```diff
@@ -266,6 +286,29 @@ const FetchInfo: React.FC<{
             );
         }
 
+        const sampleUserGuideLink =
+            PublicConfig.table_sampling?.sample_user_guide_link ?? '';
+        const sampleRateText = (
+            <div className="flex-row ml4">
+                (Full Result,
+                <span className="accent-warning-text ml4">
+                    Sampled {sampleRate}%
+                </span>
+                {sampleUserGuideLink ? (
+                    <IconButton
+                        className="ml4"
+                        onClick={() => {
+                            window.open(sampleUserGuideLink);
```

Comment:
does it need target blank to open in new tab?

Reply:
It worked new tab for me without it, but I added target blank in case